scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/task/internal/InternalTask.java

```diff
@@ -1199,6 +1203,7 @@ public synchronized void updateVariables(SchedulingService schedulingService) {
 
             updatedVariables.putAll(getSystemVariables());
         }
+        return taskResults;
```

Comment:
You never add anything to this map?

Reply:
si,,, just a few lines above,,, inside the loop....line 1194